### ISSUE TITLE
Shop not opening

### ISSUE THREAD

**Issue by Keagan-Hainsworth** [7erra/VASS-Virtual-Arsenal-Shop-System#28]
[BLACKOUT%20SECURITY%20TRAINING%20SITE.Stratis.zip](https://github.com/7erra/VASS-Virtual-Arsenal-Shop-System/files/3324902/BLACKOUT.20SECURITY.20TRAINING.20SITE.Stratis.zip)

[Arma 3 Preset Blackout Security.zip](https://github.com/7erra/VASS-Virtual-Arsenal-Shop-System/files/3324906/Arma.3.Preset.Blackout.Security.zip)

**Comment by 7erra** [comment]
Yeah the VASS folder and the description.ext are missing. Just download the content from the main branch here on GitHub and add them to the mission folder.
